cms/templates/js/signatory-details.underscore

```diff
@@ -0,0 +1,20 @@
+<% console.log("signatory-detail.underscore.start") %>
+     <div class="signatory-panel-default">
+       <div class="actions certificate-actions signatory-panel-edit">
+        <span class="action action-edit-signatory">
+            <a href="javascript:void(0);" class="edit-signatory"><i class="icon fa fa-pencil"></i> <%= gettext("Edit") %></a>
```

Comment:
Just a note that we'll need to support cancelling the edit view (back to details view) after clicking the Edit link.